models/ed/R/model2netcdf.ED2.R

```diff
@@ -82,17 +82,18 @@ model2netcdf.ED2 <- function(outdir, sitelat, sitelon, start_date,
   year_check <- unique(unlist(ylist))
   if (max(year_check) < end_year) {
     PEcAn.logger::logger.info("Run failed with some outputs.")
-    rundir <- gsub(file.path("out"), file.path("run"), outdir)
+    run_id <- basename(outdir)
+    workflow_dir <- dirname(dirname(outdir))
+    rundir <- file.path(workflow_dir, "run", run_id)
     readme <- file(file.path(rundir, "README.txt"))
     runtype <- readLines(readme, n = 1)
-    close(readme)
```

Comment:
should the file be closed?

Reply:
Ah shoot, yes it should -- I missed that it was opened via `file`. Though I'm not sure there's any benefit in opening and reading the file in separate steps, so I'll probably change this to just open and read the file in one step.